lib/AST/Attr.cpp

```diff
@@ -557,15 +557,17 @@ bool DeclAttribute::printImpl(ASTPrinter &Printer, const PrintOptions &Options,
   // SWIFT_ENABLE_TENSORFLOW
   case DAK_Differentiable: {
     Printer.printAttrName("@differentiable");
-    Printer << '(';
     auto *attr = cast<DifferentiableAttr>(this);
     auto parsedParams = attr->getParsedParameters();
-
+    // If no attribute parameter is specified, do not print parentheses at all.
```

Comment:
I wonder if it's still possible to print the canonical `wrt` parameters?
Do you prefer to hide the `wrt` parameters?

Reply:
I prefer to hide wrt parameters, as implied in the PR description. For layer APIs, when the protocol requirement says `@differentiable`, it's pretty bad style to ask the user to write `@differentiable(wrt: (self, input))`.